extern/diskann/DiskANN/src/disk_utils.cpp

```diff
@@ -1039,29 +1039,6 @@ void create_disk_layout(const T *data, uint32_t npts, uint32_t ndims, const std:
         std::ifstream reorder_data_reader;
 
         uint32_t npts_reorder_file = 0, ndims_reorder_file = 0;
-        if (reorder_data_file != std::string(""))
```

Comment:
Why remove reorder logic?

Reply:
This is the original reorder logic of DiskANN, which we did not rely on.